recipes/r-bsvars/meta.yaml

```
@@ -0,0 +1,114 @@
+{% set version = '3.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-bsvars
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/bsvars_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/bsvars/bsvars_{{ version }}.tar.gz
+  sha256: 875614c640a245c852a2fd8558618e2818b4487989a3cb6f83bf376f48076842
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # 
+
+requirements:
+  build:
+    - {{ stdlib('c') }}  # [not win]
+    - {{ stdlib('m2w64_c') }}  # [win]
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-gigrvg
+    - r-r6
+    - r-rcpp >=1.0.7
+    - r-rcpparmadillo
+    - r-rcppprogress >=0.1
+    - r-rcpptn
+    - r-stochvol
```

Needs blas and lapack

```suggestion
    - r-stochvol
    - libblas
    - liblapack
```

---

if BLAS is needed do I need to remove it from the missing DSO whitelist?

---

No, it is still required. Windows build will still link to the Rblas/Rlapack, but that should itself dynamically link to whatever implementation the user chooses to install at runtime.